src/uilib/RenderableIntel.java

```diff
@@ -13,6 +13,11 @@
 @Log4j
 public abstract class RenderableIntel extends BaseIntelPlugin {
 
+    protected String getName() {
```

Comment:
It's an abstract class, so no need to implement it here. Let the actual implementator handle it? Unless you want a safe failback...

Reply:
That was the plan, but Instead I put it on `IntelBasePlugin` which is inherited by intel classes.
`RenderableIntel` is used by board classes as well, but boards don't seem to care about `getName` (underlying starsector functionality is not using it like in case of intel with map). Tried click through it all and seems to work ok.